### PR TITLE
Archive accessibility

### LINKED ISSUE #229: Workshop archive needs accessibility improvements
* [x] Quarter headings and event headings are both `h2`s. This is confusing, since events are subordinate to quarters. Event headings should be `h3`, while individual sessions of the event should be `h4`.

* [x] It's not clear to accessibility technologies (AT) that tags should be treated as tags. Even though they are _visually_ displayed as pills, AT don't know that, and tags appear as merely a heap of words.
   * [x] We need to add `role="list"` to containers of a collection of tags, and `role="listitem"` to each tag.
   * [x] We need to add `aria-label="Tags"` to containers of a collection of tags.

* [x] The "launch" button next to each event heading is an HTML `label` element. The `label` element should only be used in forms, not for a link. 

   image

- [x] The accordion buttons have a weird label. We should make sure neither "Launch" nor "javascript" is included in the label for that button.

   image

* [x] It's unclear to AT what the `div` for each event is. As far as it sees, the `div` is just a generic container. We need to add `aria-label` or `aria-labelled-by` with the title of the event.

   image

### PR DESCRIPTION
**Pull request by jamesmwu**
# Changes

Updated event heading and session header tags, added roles/labels to pills (collection of tags and tags themselves), removed label HTML tag from Launch Link, updated AT labels for events. Fixes 229, makes archive page more accessible for AT

## Type of change

- [X] Enhancement 

# Related Issues

Fixes #229

### CODE REVIEW

**Review by katelynsyu — APPROVED** [review]
Lgtm! Thanks!

File: README.md [hunk 1]
```diff
@@ -12,12 +12,14 @@ This repository contains the code for ACM Hack’s website!
 This website is developed with [Gatsby](https://www.gatsbyjs.org/) and
 [Material UI](https://material-ui.com/), and is an active project being
 developed by ACM Hack’s Dev Team:
-Alex Xia [(@khxia)](https://github.com/khxia/),
-Jody Lin [(@jodymlin)](https://github.com/jodymlin/),
 Nareh Agazaryan [(@nareha)](https://github.com/nareha),
+Katelyn Yu [(@katelynsyu)](https://github.com/katelynsyu),
 Einar Balan [(@EinarBalan)](https://github.com/EinarBalan),
 Timothy Gu [(@TimothyGu)](https://github.com/TimothyGu/),
```
Comment (TimothyGu) [inline README.md:18]: :wave: Please feel free move me (and other folks) to a separate "alumni" section or something like that :) I'm clearly not part of the ACM Hack Dev Team anymore.

Comment (jamesmwu) [reply README.md]: You'll always be a part of dev team in our hearts 🥹

**Review by katelynsyu — APPROVED** [review]
Lgtm!

**Review by nareha — APPROVED** [review]
lgtm, thank you for picking this up! 💜

**Pull request merged**